### ISSUE TITLE
[BUG] xmldb:login does not log-in to a remote collection over xmlrpc

### ISSUE THREAD

**Issue by stexandev** [eXist-db/exist#3767]
**this is work in progress** I am going to further explain this issue in the next days, it may be related to ...

**Describe the bug**
When using xmldb:login on a remote collection with valid credentials, the function returns true but does not actually log in the user. it seems the user will always be unauthenticated, i.e. "guest"

**Expected behavior**
the user is logged in on the remote collection an can create collections/resources according to the rights granted to him

**To Reproduce**
suppose, you are running existdb on the host "app" on port 8080

```Xquery
xquery version "3.1";

xmldb:login("xmldb:exist://app:8080/exist/xmlrpc/db", "admin", ""),
sm:id(),
sm:is-authenticated(),
sm:is-externally-authenticated(),

xmldb:create-collection("xmldb:exist://app:8080/exist/xmlrpc/db/apps", "data")
```

**Context (please always complete the following information):**
 - existdb/existdb:5.2.0
 - Docker version 19.03.8, build afacb8b7f0
 - Ubuntu 20.04

**Additional context**
- Docker

**Comment by adamretter** [comment]
@stexandev You should not rely on the XMLDB XQuery Module for access to remote systems.

**Comment by stexandev** [comment]
@adamretter but what should I rely on alternatively to access remote collections from an xquery app?

**Comment by stexandev** [comment]
I would like to point out, that it would be nice to know whether this is considered a bug that will be fixed or an expected behavior with wrong/missing user feedback. If `xmldb:login` to a remote collection is considered a feature, it should be made working; if not, it should at least raise an error that logging in to a remote collection ins not within the scope of the function.

**Comment by adamretter** [comment]
@stexandev I am not sure there is an alternative apart from plain HTTP via the REST Server at present.

**Comment by line-o** [comment]
I agree with @stexandev that if it is not considered to work it should throw.
